src/utils/unrolled_lstm.py

```diff
@@ -0,0 +1,167 @@
+import math
```

Comment:
Just to confirm, this is the exact same copy as that from the forecaster repo?

Reply:
yes